.ci_support/compute_build_graph.py

```diff
@@ -0,0 +1,597 @@
+#!/usr/bin/env python
```

Comment:
This is pretty big to vendor.  It's probably OK, but it would be good to re-examine this vendoring decision in a few months and see if it's still a good idea, or if it would be better to add a dependency on c3i.

Reply:
I prefer adding c3i as a dependency, but c3i is not on defaults, so I thought it was meant for internal use only.

Reply:
There's also a lot of unused stuff here.  I'm not sure if you want to remove it, or just leave it in for easier diffs down the line, should there be changes to both c3i and this file.

Reply:
We can work on getting it onto defaults.  It has only seen internal use, but I wouldn't say it's only meant for internal use.  It's just much more oriented towards our needs, but could grow more general if there's community use and input (PRs definitely welcome).

Reply:
Sorry to be late. What was the outcome on this?

Reply:
https://github.com/conda/conda-concourse-ci/issues/71#issuecomment-369421274

Reply:
This functionality will be split from c3i and make its way into conda-build.  No timeline right now, but hopefully soon.  I don't think it will take too much work to have it simply work.  Having it go fast will take more time.